quiz.js

```diff
@@ -0,0 +1,104 @@
+
+// A key that specifies which character to use to build the pine tree.
+
+// The character to use should be from user input in a <input type="text"> field in the DOM.
+
+// Once the user enters in a number, and a character, the user can either then just
+// press the enter key (as long as the cursor is in one of the input fields),
+// or click a button that is labeled "Grow your tree" and the tree should be shown in the console.
+// This requires you to add an event listener to the button, as well as an event listener for the enter/return key.
+
+// If either of the input fields does not have a value in it when the user presses the enter key,
+// or presses the button, then display an alert stating that both fields must have a value.
+
+// Grow your tree
+
+// Example
+
+// Here's what the pine tree should look like when you specify a height of 7, and use the asterisk character.
+
+//       *                spaces= 6, symbol= 1  (0) + 1
+//      ***               spaces= 5, symbol= 3  (1) + 2
+//     *****              spaces= 4, symbol= 5  (2) + 3
+//    *******             spaces= 3, symbol= 7  (3) + 4
+//   *********            spaces= 2, symbol= 9  (4) + 5
+//  ***********           spaces= 1, symbol= 11 (5) + 6
+// *************          spaces= 0, symbol= 13 (6) + 7
+
+
+
+// var height = document.getElementById("treeHeight").value;
+// console.log("height:", height);
+
+// var char = document.getElementById("character").value;
+// console.log("character:", char);
+
+// var tree = "";
+// console.log("tree:", tree);
+
+
+
+var heightBox = document.getElementById("treeHeight");		// target input box in HTML, make into a variable
+heightBox.addEventListener("keypress", enter);						// add an event listener that triggers
+																													// the function "enter" upon a keypress
+
+var charBox = document.getElementById("character");
+charBox.addEventListener("keypress", enter);
+
+var button = document.getElementById("submitButton");		// target the "grow" button and assign it to a variable
+button.addEventListener("click", middle);								// create an event listener that, upon the click of the button variable, runs the function "middle"
+
+
+
+function enter (e) {															// function that says "if the keyCode of the key thats pressed is 13,
+	var key = e.keyCode;														// run the function 'middle'
+	if (13 === key)	{
+		middle();
+	}
+}
+
+var treeObject = {};															// create an empty object called "treeObject"
+
+function middle () {
+	treeObject.height = heightBox.value;						// create a key in treeObject called "height" and add to it the text in the height input box
+	treeObject.character = charBox.value;						// create a key in treeObject called "character" and add to it the text in the character input box
+	buildTree(treeObject)														// run the function "buildTree" with the treeObject object
+	console.log("height:", treeObject.height);
```

Comment:
As per the instructions, the output should be just the tree itself.  These informative logs and the object log at line 98 should be removed for final submission (though logging them during production for troubleshooting is an _awesome_ idea)